src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -1044,6 +1032,28 @@ public static nuint CommonPrefixLength(ref byte first, ref byte second, nuint le
             return i + uint.TrailingZeroCount(mask);
         }
 
+#if MONO
+        // Vector sub-search adapted from https://github.com/aspnet/KestrelHttpServer/pull/1138
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static int LocateFirstFoundByte(Vector<byte> match)
```

Comment:
this looks to be a sub-optimal implementation (not using the MoveMask/TLZ trick like we do for V128/V256) but since we plan to get rid of it I guess it doesn't matter

Reply:
I've not changed any code. I only moved a few methods in the source to reduce the number of ifdefs.